theano/gpuarray/blas.py

```diff
@@ -1726,9 +1735,14 @@ def local_inplace_gpuagemm(node, inputs):
 def local_inplace_gpuager(node, inputs):
     return [gpuger_inplace(*inputs)]
 
+@inplace_allocempty(GpuGemmBatch, 0)
+def local_inplace_gpuagemmbatch(node, inputs):
```

Comment:
This opt would need a simple test. I think current test won't fail if we disable this opt.